README.md

```diff
@@ -110,7 +110,7 @@ you will get TypeScript typings compatible with a given library.
 
 ### Step by step guide
 
-Install TypeChain with `yarn add --dev typechain` and install desired target.
+Install TypeChain with `pnpm install --save-dev typechain` and install desired target.
```

Comment:
I'd keep yarn here, users don't need to know that we use pnpm

Reply:
True. This might be a separate matter,.

What package manager should we refer to in our user-facing readmes?
Should we just use `npm` here as it's the _"default"_?

Reply:
yeah maybe we should rewrite it to simply use `npm`